main.py

```diff
@@ -1,10 +1,54 @@
 import init_django_orm  # noqa: F401
-
+import json
+from django.utils import timezone
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as file:
+        players_data = json.load(file)
+
+    for player_name, player_data in players_data.items():
+        print(player_data)
+
+        race_name = player_data["race"]["name"]
+        race_description = player_data["race"].get("description", " ")
+
+        race, created = Race.objects.get_or_create(
```

Comment:
Since you do not use `created`, you could omit it here and in the other places
```suggestion
        race, _ = Race.objects.get_or_create(
```